scripts/get-latest-platform-tests.js

```diff
@@ -14,7 +14,7 @@ const request = require("request");
 // 1. Go to https://github.com/w3c/web-platform-tests/tree/master/url
 // 2. Press "y" on your keyboard to get a permalink
 // 3. Copy the commit hash
-const commitHash = "3afae94fd4c945486a165e141a7755aa2a4908a3";
+const commitHash = "cfd901b9c07d5f43fcbd0464815d9e26bd48f365";
```

Comment:
Wow, I have no idea how this works given that it's a commit hash from rmisev/web-platform-tests and not from w3c/web-platform-tests, but it seems to work.